CHANGELOG_PENDING.md

```diff
@@ -22,6 +22,7 @@
 
 #### General
 - \#2299 Split devtool Orchestrator run scripts into versions with/without external transcoder and prevent Transcoder/Broadcaster run scripts from using same CLI port (@thomshutt)
+- \#2334 Fix issue with checksum generation step with branch names having a `/` (forward slash) character (@hjpotter92)
```

Comment:
I'd probably not add GH Action changes to the changelog, since AFAIU CHANGELOG is user facing, so everything we release for users.

Reply:
Thanks! removed the same.